### ISSUE TITLE
CopyAs not displayed when token is selected

### ISSUE THREAD

**Issue by bb33bb** [Vector35/binaryninja-api#5910]
**What is the feature you'd like to have?**
select code from top to down ,then press right button , we can see the menu of
Copy As
but if we select code from bottom to top, that menu will not display or will display, dont know why
**Is your feature request related to a problem?**
NO

**Are any alternative solutions acceptable?**
Dont know

**Additional Information:**

https://github.com/user-attachments/assets/bd06642e-1674-4db7-9813-ed0407ff811c

**Comment by xusheng6** [comment]
Yeah it looks like a bug. For context, we would decide to show or not show certain menu items based on the selection. In the case showed in the video, it appears we are making the decision wrong at certain times. ~~@bb33bb could it be possible to share with us a binary and also a range of selection that can reproduce this?~~

Update: no worries, we can reproduce it already

**Comment by emesare** [comment]
While waiting for this to be fixed, you should be able to select the space to the right of the tokens, it seems that clicking on the token will override the selection and not show the "Copy As" commands.

https://github.com/user-attachments/assets/9855fe3e-2593-4fa1-b69f-957211424571
